tools/rapids-download-wheels-from-github

```diff
@@ -8,7 +8,6 @@ set -eo pipefail
 export RAPIDS_SCRIPT_NAME="rapids-download-wheels-from-github"
 
 source rapids-prompt-local-repo-config
-source rapids-prompt-local-github-auth
```

Comment:
Removing this because this script doesn't directly call the `gh` CLI. These `rapids-prompt-local-github-auth` calls should only be placed in scripts that directly need GitHub auth, to minimize unnecessary calls.